libecho/include/EchoProtocol.h

```diff
@@ -42,42 +62,72 @@ class EchoProtocol {
 
     /**
      * @brief Closes a connection.
+     * Connection that was once opened should be closed (even when one of read/write
+     * functions throws an exception).
      */
     void close();
 
     /**
-     * Reads at most count bytes from audio input.
-     * Can receive 0 bytes, but if it doesn't receive any signal for timeout seconds,
-     * it throws an exception.
+     * @brief           Reads at most @p count bytes from audio input and fills buffer with them.
+     * @param buf       pointer to buffer that should be filled
+     * @param count     number of bytes to read
+     * @param timeout   number of seconds; if no signal is detected by then, function assumes
+     *                  that connection is broken; some signals should be continuously sent
+     *                  by protocol to keep connection alive even when no data is being sent;
```

Comment:
"should be continuously sent by protocol" - to jest niejasne. Czy to znaczy, że użytkownik powinien coś wysyłać, czy że protokół automatycznie wysyła te keep alive? Jesli protokół to robi, to czemu "should be", a nie "are"?